pkg/lightning/backend/local/local.go

```diff
@@ -1477,7 +1477,7 @@ func (local *Backend) doImport(
 
 	if local.ticiWriteGroup != nil {
 		// If the import is done, we can close the write group.
-		return local.ticiWriteGroup.MarkTableUploadFinished(ctx)
+		return local.ticiWriteGroup.FinishIndexUpload(ctx)
 	}
```

Comment:
@OliverS929 If there are multiple DXF instances to run the `IMPORT INTO` task, will this API called only once in the whole `IMPORT INTO` task? Or the API will be called by each DXF instance once?